scheduler/ecs/ecs.go

```diff
@@ -78,6 +78,12 @@ type Config struct {
 
 	// AWS configuration.
 	AWS client.ConfigProvider
+
+	// Log driver to use when starting ECS tasks. Maps to the --log-driver docker cli arg
+	LogDriver string
+
+	// Log options to use when starting ECS tasks. Maps to the --log-opt docker cli arg
+	LogOpts []string
```

Comment:
I think we should probably do the `foo=bar` parsing earlier (in cmd/empire) and take a `map[string]string` here instead.

Actually, maybe even just make this `LogConfiguration *ecs.LogConfiguration` and build the LogConfiguration in cmd/empire.